test/functional/JIT_Test/src/jit/test/jitserver/JITServerTest.java

```diff
@@ -46,6 +50,7 @@ public class JITServerTest {
 	private static final int PROCESS_TERMINATE_WAIT_TIME_MS = 2 * 1000;
 	private static final int SERVER_START_WAIT_TIME_MS = 5 * 1000;
 	private static final int CLIENT_TEST_TIME_MS = 45 * 1000;
+	private static final int CLIENT_TEST_TIME_MS1 = 15 * 100;
```

Comment:
I see that this is used after destroying the cache, so maybe we should call is something suggestive like, `DESTROY_SCC_WAIT_TIME_MS`. Also, there is no reason to use `15 * 100`; use 1500 directly. Multiplication with 1000 made sense because we were transforming from sec to millseconds, but multiplying by 100 does not have any special meaning.

Reply:
Done.